saleor/graphql/giftcard/bulk_mutations.py

```diff
@@ -151,6 +152,14 @@ class Meta:
         permissions = (GiftcardPermissions.MANAGE_GIFT_CARD,)
         error_type_class = GiftCardError
 
+    @classmethod
+    def clean_instance(cls, info, instance):
+        if is_gift_card_expired(instance):
+            raise ValidationError(
+                "Cannot activate expired card.",
```

Comment:
Shouldn't be connected with a proper `field`? (Maybe it is already handled in place where this method is called)

Reply:
Something like this:
```
        raise ValidationError(
             {
                 "id": ValidationError(
                     "Expired gift card can only be deleted.",
                     code=GiftCardErrorCode.EXPIRED_GIFT_CARD.value,
                 )
             }
         )
```

Reply:
This is handled in `Base` mutation and adding the `id` does not change anything as in `Base` as a field the instance `id` is set.